yellr-serv/yellrserv/templates/new/assignments.mak

```diff
@@ -0,0 +1,31 @@
+<%inherit file="base.mak"/>
+
+<div class="row">
+  <div class="large-12 columns">
+    <div class="tab-links">
+      <div class="tab-link-container"><a class="tab-link top-link-divider" href="/local">Local</a></div>
+      <div class="tab-link-container selected-tab"><a class="tab-link top-link-divider" href="/assignments">Assignments</a></div>
+      <div class="tab-link-container"><a class="tab-link top-link-divider" href="/stories">Stories</a></div>
+      <div class="tab-link-container"><a class="tab-link" href="/post">Post</a></div>
+    </div>
+    <hr/>
+  </div>
+</div>
+
+<div class="row">
+  <div class="medium-10 medium-centered columns">
+    % for assignment in assignments:
+    <div class="container-box">
+      <div>
+        <i class="fa fa-comments right"> ${assignment['post_count']}</i>
```

Comment:
I don't think that you should be placing content inside of an `<i>` tag. This tag is specifically for the icon itself and content associated with it should just be places next to it. I think this is causing some issues with fonts because this is what the font looks like on my box.

![screenshot 2015-04-22 12 53 30](https://cloud.githubusercontent.com/assets/2600677/7280234/5d1e0d22-e8ef-11e4-8492-97f3e3923bf8.png)